bouncer/src/repo/lib/datastructure/repo_structs.h

```diff
@@ -104,6 +104,7 @@ typedef struct{
 	float opacity = 1;
 	float shininess = 0;
 	float shininessStrength = 0;
+	float lineWeight = 0;
```

Comment:
I think we either need to ensure default lineWeight is one, or it defaults to NaN (so it's not written)

Otherwise other processes which doesn't read line weight (like the ODA importer) will stop seeing lines when we take this property into account!